src/qservkafka/models/kafka.py

```diff
@@ -175,6 +179,20 @@ def is_string(self) -> bool:
         """Check whether the underlying data type is a string."""
         return self.datatype.is_string()
 
+    def is_array(self) -> Literal[0] | bool | None:
+        """Determine if the column is an array.
+
+        Returns
+        -------
+        bool
+            True if the column is an array otherwise False.
+        """
+        if not self.arraysize:
+            return False
+        if self.arraysize.variable:
+            return True
+        return self.arraysize.limit and self.arraysize.limit > 1
```

Comment:
```suggestion
        return bool(self.arraysize.limit) and self.arraysize.limit > 1
```
should let you get rid of `Literal[0]` in the type signature.

Reply:
Slightly changed this to check that limit is not None, hope that's ok:
`return self.arraysize.limit is not None and self.arraysize.limit > 1`